packages/flutter_test/lib/src/binding.dart

```diff
@@ -1227,23 +1227,39 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
 ///
 /// These values are set on the binding's
 /// [LiveTestWidgetsFlutterBinding.framePolicy] property. The default is
-/// [fadePointers].
+/// [fadePointers]. To do set a value while still allowing the test file to
```

Comment:
`To do set` -> ` To set`?

Nit: `a normal test` -> `a normal unit test`, and add `{}` to the `if` in the sample code?

Reply:
> Nit: `a normal test` -> `a normal unit test`

I think it's not necessarily a unit test. This is more often used for integration and performance test.

Reply:
Done other than the above.

Reply:
@CareF : your `git push` probably failed as Github still shows your commit 6 days ago as the last commit.

Reply:
> @CareF : your `git push` probably failed as Github still shows your commit 6 days ago as the last commit.

My bad! I pushed again. @liyuqian